blob/driver/driver.go

```diff
@@ -208,32 +208,15 @@ type Bucket interface {
 	// one of the other methods in this interface.
 	ErrorCode(error) gcerrors.ErrorCode
 
-	// As allows providers to expose provider-specific types.
-	//
-	// i will be a pointer to the type the user wants filled in.
-	// As should either fill it in and return true, or return false.
-	//
-	// Mutable objects should be exposed as a pointer to the object;
-	// i will therefore be a **.
-	//
-	// A provider should document the type(s) it support in package
-	// comments, and add conformance tests verifying them.
-	//
-	// A sample implementation might look like this, for supporting foo.MyType:
-	//   mt, ok := i.(*foo.MyType)
-	//   if !ok {
-	//     return false
-	//   }
-	//   *i = foo.MyType{}  // or, more likely, the existing value
-	//   return true
-	//
-	// See
-	// https://github.com/google/go-cloud/blob/master/internal/docs/design.md#as
-	// for more background.
+	// As converts i to provider-specific types.
```

Comment:
See my comment below about the docstring for driver packages.

Reply:
Done